### ISSUE TITLE
Batch

### ISSUE THREAD

**Issue by techtangents** [TicketSwap/target-redshift#131]
Does this target support batch messages?

**Comment by tobiascadee** [comment]
I have not looked into the feature yet, but I think it does not require any additional configuration on the target side to work?

**Comment by edgarrmondragon** [comment]
> I have not looked into the feature yet, but I think it does not require any additional configuration on the target side to work?

Yeah, although by default the SDK only handles JSONL batches, and other encodings such as parquet or CSV are left the target to implement.

**Comment by tobiascadee** [comment]
then I can look into it in the near feature!
